### ISSUE TITLE
Spaces removed when needed around > or <

### ISSUE THREAD

**Issue by Ruckt** [nicklockwood/SwiftFormat#729]
Format changes
``` 
} else if sizeCategory < .accessibilityLarge,
            sizeCategory > .extraExtraLarge
{
```
to
``` 
 } else if sizeCategory<.accessibilityLarge,
            sizeCategory>.extraExtraLarge
{
```

Spaces around > and < removed and thus doesn't compile.

**Comment by nicklockwood** [comment]
@Ruckt thanks for reporting this - what format configuration options are you using?

**Comment by Ruckt** [comment]
I'm using the default format. Haven't changed anything.

**Comment by nicklockwood** [comment]
@Ruckt ah, OK. It looks like a bug in the parser - it's mistaking the < and > for generic arguments. I'll fix it, but you can work around it for now by adding the following to the top of the file:

```swift
// swiftformat:disable spaceAroundGenerics spaceInsideGenerics spaceAroundOperators wrapArguments
```

**Comment by nicklockwood** [comment]
@Ruckt fixed in 0.46.2.
